submissions/sprint_c/Pipfile

```diff
@@ -0,0 +1,15 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[dev-packages]
+
+[packages]
+python-dotenv = "*"
+flask = "*"
+flask-testing = "*"
```

Comment:
flask-testing shouldn't be installed in production environments and therefore shouldn't be included in the packages section. The package belongs in dev-packages.